cargo-concordium/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased changes
 
+## 2.3.0
+
+- Add support for V3 schemas that include support for event schemas. This enables
+  `cargo-concordium` to build and interact with smart contracts using
+  `concordium-std` version 4.1.
```

Comment:
I think this needs to change a bit. The version will be 2.4.0 now since 2.3.0 is already released.